.github/workflows/main.yml

```diff
@@ -381,6 +381,18 @@ jobs:
           df -h
           docker system df
 
+  # google chat notification about release
+  notification:
+    name: Google chat notification about release and published version
+    needs: [publish]
+    runs-on: ubuntu-latest
+    if: github.event_name == 'release' && startsWith(github.ref, 'refs/tags')
+    steps:
+      - name: Send notification to google chat room "DSP releases"
+        uses: lakto/google-chat-action@main
```

Comment:
what is this?

Reply:
do you mean the package? it's the google chat notification action (forked from another github actions, because I had to change something)